### DISCUSSION TITLE
Mask that  feature the burbles in the ThunderKAT observation

### DISCUSSION THREAD

**Discussion by Victoria-Samboco** [ratt-ru/solarkat#32] [General]
I run breizorro with the option ```number-islands``` whit the command ```breizorro -r im9-MFS-image.fits  --number-islands -o mask_islands```  but I don't know how to check the island numbers. I expected to see it in the image, but I was wrong. Could you please help me how to see the island numbers?

![image](https://github.com/ratt-ru/solarkat/assets/95472901/4be28145-d1db-4fe0-a719-f3ad46e8cae6)

As an alternative, in the meantime, I also used another option where I created manual ds9 regions from CARTA and then merged with a five threshold mask (where the artefacts do not exist) to generate a mask :

![image](https://github.com/ratt-ru/solarkat/assets/95472901/3e573b45-fcf1-4de7-9f11-a913035cefe0)

**Comment by o-smirnov** [comment]
> Could you please help me how to see the island numbers?

@Victoria-Samboco open the renumbered mask in CARTA, move the cursor over the island, and read off the pixel value at the top of the screen.

**Comment by Victoria-Samboco** [comment]
Alright, I got it. Thanks

**Comment by Victoria-Samboco** [comment]
I identified the islands in the extended emission (left image). I merged them into a mask with a higher threshold resulting in the right image (but the result was still not satisfactory). 

![image](https://github.com/ratt-ru/solarkat/assets/95472901/b2fe9204-dd38-48ba-b717-25db01512413)

The extended emission is still there (right bottom image). Now I am trying another. Experiment, which is cleaning only that region with the multiscale clean option ( then I will only clean the rest), but it has been running since yesterday already. I am giving it a try. I don't know if it is gonna work.
![image](https://github.com/ratt-ru/solarkat/assets/95472901/8d7974e3-8507-48cb-9dc9-b9a8af93b6cc)
